README.md

```diff
@@ -55,7 +55,7 @@
      | OrderingFields | It enables fields ordering in the raw message by dictionary (when changing fields numbers) | true / false | |
      | FakeResendRequest | Response to ResendRequest(2) by heartbeats | true / false | false |
      | EnableNextExpectedMsgSeqNum | Add tag NextExpectedMsgSeqNum (optional tag 789) on the sent Logon message and use value of tag 789 on received Logon message to synchronize session. This should not be enabled for FIX versions < 4.4. Only works when ValidateSequenceNumbers is enabled. | true / false | false |
-     | TimeZone | Time zone for this session; if specified, the session start and end will be converted from this zone to UTC. | Time zone ID (America/New_York, Asia/Tokyo, Europe/London, etc.) ||
+     | TimeZone | Time zone for this session; if specified, the session start and end will be converted from this zone to UTC. | Time zone ID (America/New_York, Asia/To kyo, Europe/London, etc.) ||
```

Comment:
> Asia/To kyo

It would be better just to remove it altogether. We got sensitive data (not sure why `Tokyo` is sensitive data though) checks working? Why it wasn't firing before?

Reply:
I asked Dmitry Machrovsky earlier about deleting this word. I'll check it out now